pyproject.toml

```diff
@@ -10,3 +10,39 @@ line_length = 88
 profile = 'black'
 sections = ['FUTURE', 'STDLIB', 'THIRDPARTY', 'EXTERNAL_PACKAGE', 'FIRSTPARTY', 'LOCALFOLDER']
 skip_glob = ['download_and_create_reference_datasets/*', 'gcloud_dataproc/*', 'hail_scripts/*', 'kubernetes/*']
+
+[tool.ruff]
+extend-exclude = [
```

Comment:
Happy to have this run on everything I touch! Out of scope here, but like we briefly discussed, I made ticket #441 for the precommit hook